python/kserve/kserve/handlers/http.py

```diff
@@ -65,20 +66,68 @@ def get_model(self, name: str):
 
 
 class PredictHandler(HTTPHandler):
+    def create_cloudevent(self, model_name: str, body: Union[Dict, CloudEvent], response: Dict,
+                          binary_event=False) -> Dict:
+        ce_attributes = {}
+
+        if os.getenv("CE_MERGE", "false") == "true":
```

Comment:
Maybe use the `bool()` function for comparison rather than comparing strings?

Reply:
Do you mean like `bool(str)` ? This returns True for any non empty string, and False for empty string.

I've added `.lower()` to help match more cases